v2/fleet-local/control/marathon@.service

```diff
@@ -21,10 +21,14 @@ ExecStartPre=-/usr/bin/docker rm marathon
 ExecStart=/usr/bin/sh -c "/usr/bin/docker run \
   --name marathon \
   -e LIBPROCESS_PORT=9090 \
+  -e marathon_user=`etcdctl get /marathon_user` \
+  -e marathon_password=`/etcdctl get marathon_password` \
```

Comment:
might be a good idea to scope these, i.e
``-e MARATHON_USER=`etcdctl get /FD/MARATHON_USER` \``